File: X/clang/XRPL_UKL1/soal2.c
/*
Soal    : 2
a       : 5
b       : 6
*/

#include <stdio.h>

int main(){
    int a=5,
        b=9,
        u=a,
        s=a,
        total=0;
    printf("Deret aritmatikanya adalah\n"); 
    for(int i=0;i<5;i++){                            
        for(int j=0;j<6;j++){  
            printf("%d\t", u); 
            u+=b;
            s+=u;
        }
        printf("\n");
    }
    total=s-u;
    printf("Jumlah Deret aritmatikanya adalah %d\n", s); 
}

File: X/typescript/soal2.ts
/*
Soal    : 2
a       : 5
b       : 6
*/

let a = 5,
  b = 9;
let u = a,
  s = a;

console.log("Deret aritmatikanya adalah");

for (let i = 0; i < 5; i++) {
  for (let j = 0; j < 6; j++) {
    process.stdout.write(u + "\t");
    u += b;
    s += u;
  }
  console.log();
}

let total = s - u;
console.log("Jumlah deret aritmatikanya adalah " + s);
